velox/exec/TaskStats.h

```diff
@@ -55,6 +55,9 @@ struct TaskStats {
   int32_t numQueuedSplits{0};
   std::unordered_set<int32_t> completedSplitGroups;
 
+  // Barrier stats.
```

Comment:
Would you clarify a bit? Is this the total number of completed barriers or does this also include in-progress barrier? Do we need to include "Task" in the name? I noticed that the same term, barrier, is also used for waiting for hash build operator to finish processing. Do we expect any confusion to arise from using the same term in different contexts?

Reply:
@xiaoxmeng 

> We don't use barrier for hash build anymore.

I see. Then, what are the barriers used in Task::allPeersFinished?

```
  const auto splitGroupId = caller->driverCtx()->splitGroupId;
  auto& barriers = splitGroupStates_[splitGroupId].barriers;
  auto& state = barriers[planNodeId];
```